plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -27,44 +27,43 @@
 /**
  * Callback for WordPress 'the_content' filter.
  *
- * @param string $content The post content.
+ * @param null|string $content The post content.
  *
- * @return string The post content.
+ * @return null|string The post content.
  */
-function content_replacement( string $content ): string {
-
-	if ( ! $content ) {
-		return '';
-	}
-
-	$replace_content_urls = domain_replacement_enabled();
-	$replace_media_urls   = ! use_wp_domain_for_media();
+function content_replacement( $content ) {
```

Comment:
Can't you just do `$content = ''`?

Reply:
Updated. Thanks @theodesp for that suggestion.